src/main/java/QueueNumber.java

```diff
@@ -0,0 +1,30 @@
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class QueueNumber {
+    Queue<Number> numberQueue = new LinkedList<>();
```

Comment:
Mọi thành viên đều phải có access modifer nha

```suggestion
    private Queue<Number> numberQueue = new LinkedList<>();
```

Reply:
Mình đang implement Queue nên ko dùng cái có sẵn nha. Chỉ dùng arrayList hoặc list